mlir/lib/Dialect/Bufferization/Transforms/BufferUtils.cpp

```diff
@@ -11,6 +11,8 @@
 //===----------------------------------------------------------------------===//
 
 #include "mlir/Dialect/Bufferization/Transforms/BufferUtils.h"
+
```

Comment:
is the extra blank line preventing clang-format from sorting the headers.

Reply:
I put it in there manually. In a file `foo.cpp`, we usually have an empty like between `include "foo.h"` and the remaining headers.